gm4_mob_curing/beet.yaml

```diff
@@ -13,7 +13,7 @@ meta:
   gm4:
     versioning:
       required:
-        lib_trades: 1.3.0
+        lib_trades: 1.4.0
```

Comment:
This is a problem. The `lib_trades` version is still 1.3.X and the following warning also shows in the log
![image](https://github.com/Gamemode4Dev/GM4_Datapacks/assets/17352009/1144821b-5278-40a9-9164-d708621438c3)